azure-mgmt-datafactory/azure/mgmt/datafactory/operations/activity_runs_operations.py

```diff
@@ -109,7 +109,7 @@ def internal_paging(next_link=None, raw=False):
             # Construct and send request
             request = self._client.get(url, query_parameters)
             response = self._client.send(
-                request, header_parameters, **operation_config)
+                request, header_parameters, stream=False, **operation_config)
```

Comment:
This one too - not sure why we see this added

Reply:
This is an optmisation of latest Autorest. TL;DR; This improve HTTP session pooling.